src/formats/skel.rs

```diff
@@ -48,3 +38,15 @@ pub struct Skel {
     pub transforms: SsbhArray<Matrix4x4>,
     pub inv_transforms: SsbhArray<Matrix4x4>,
 }
+
+#[cfg_attr(feature = "derive_serde", derive(Serialize, Deserialize))]
+#[derive(BinRead, Debug, Clone, Copy)]
+#[br(repr(u8))]
+pub enum BillboardType {
```

Comment:
It looks like there is also a variant for 3.

Reply:
In testing, 3 didn't provide any billboard functionality, so I just left it out of the enum and assumed it wasn't used. Testing those three models above also doesn't show any noticeable billboarding, so it may be an option to add it to the enum and label it as "Unused."